### PR TITLE
[FEATURE] Add types > creationOptions > defaultValues

### PR DESCRIPTION
**Pull request by linawolf**
Related: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/801
Resolves: https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TCA/issues/1161
Releases: main

### CODE REVIEW

File: Documentation/Types/_CodeSnippets/_creationOptions.php [hunk 1]
```diff
@@ -9,6 +9,9 @@
                 'creationOptions' => [
                     'saveAndClose' => true,
                 ],
+                'defaultValues' => [
+                    'bodytext' => '<p>some text</p>',
+                ],
```
Comment (froemken) [inline Documentation/Types/_CodeSnippets/_creationOptions.php:14]:
```suggestion
                'creationOptions' => [
                    'saveAndClose' => true,
                    'defaultValues' => [
                        'bodytext' => '<p>some text</p>',
                    ],
                ],
```

File: Documentation/Types/_Properties/_CreationOptions.rst.txt [hunk 2]
```diff
@@ -17,7 +17,22 @@
...
+        :Examples: :ref:`types-example-creation-option`
+
+        ..  versionadded:: 13.0
+
         If true, clicking on the new element wizard directs the user back to
         the page module directly instead of showing the edit form from the form
-        engine. Can be overridden with page TSconfig
-        :confval:`newContentElement.wizardItems.[group].elements.[name] <t3tsconfig:mod-wizards-newcontentelement-wizarditems>`.
+        engine. Can be overridden with page TSconfig option
+        :confval:`tt_content_defValues <t3tsconfig:mod-wizards-newcontentelement-wizarditems-group-elements-name-tt-content-defvalues>`.
```
Comment (froemken) [inline Documentation/Types/_Properties/_CreationOptions.rst.txt:38]: In your linked changelog I found following quote: "The option has been renamed from
`tt_content_defValues` to `defaultValues`:". 

```suggestion
        :confval:`defaultValues <t3tsconfig:mod-wizards-newcontentelement-wizarditems-group-elements-name-tt-content-defvalues>`.
```

Comment (linawolf) [reply Documentation/Types/_Properties/_CreationOptions.rst.txt:38]: Yes but the TSconfig option has not been renamed. See https://github.com/TYPO3-CMS/backend/blob/main/Classes/Controller/ContentElement/NewContentElementController.php#L386

File: Documentation/Types/_Properties/_CreationOptions.rst.txt [hunk 3]
```diff
@@ -17,7 +17,22 @@
         :default: false
         :Examples: :ref:`types-example-creation-option`
 
+        ..  versionadded:: 13.0
+
+        If true, clicking on the new element wizard directs the user back to
```
[the remark this answers is not in the document]
Comment (linawolf) [reply Documentation/Types/_Properties/_CreationOptions.rst.txt]: Thanks for you review. It actually made me realize that I had accidently copied over the description of the safe and close button. So I applied your suggestion to the afe and close button  and added a new description for the "defaultValues"

**Pull request merged**